### DISCUSSION TITLE
Difference from the zod-prisma generator?

### DISCUSSION THREAD

**Discussion by scpedicini** [chrishoermann/zod-prisma-types#81] [General]
Is there a reason that you decided to create a new prisma-to-zod generator rather than using this one or this one?
https://github.com/CarterGrimmeisen/zod-prisma
https://github.com/omar-dulaimi/prisma-zod-generator

I'm weighing options and just trying to figure out the differences. 

_EDIT: Looks like the CarterGrimmeisen project hasn't been touched in a year. The one by Omar seems more maintained (supports Prisma 4)._

**Comment by chrishoermann** [comment]
@scpedicini at the time I decided to write this package `zod-prisma` wasn't actively maintained anymore (and still isn't) and `prisma-zod-generator` lacked the possibility of adding validators directly in the `prisma schema` comments. Also `zod-prisma` didn't generate the `input-types`, `argTypes`, etc. and `prisma-zod-generator` didn't generate plain model schemas. 

So I decided to combine the features of both packages and add som DX (e.g. error messages while generation with wrong validators).
